src/pilot/assign_role.py

```diff
@@ -1475,15 +1479,17 @@ def main():
                 if not succeeded:
                     sys.exit(1)
 
-            succeeded = configure_bios(
-                node,
-                ironic_client,
-                bios_settings,
-                drac_client)
-
-            if not succeeded:
-                sys.exit(1)
+            if not args.skip_bios_config:
+                succeeded = configure_bios(
+                    node,
+                    ironic_client,
+                    bios_settings,
+                    drac_client)
 
+                if not succeeded:
+                    sys.exit(1)
+            else:
+                LOG.info("Skipping BIOS configuration")
```

Comment:
I like that you have a logging statement here when skipping BIOS configuration.  Would you mind adding a similar logging statement above when RAID configuration is skipped?

Reply:
done